Memorize/Models/MemoryGameModel.swift

```diff
@@ -72,4 +63,87 @@ struct MemoryGameModel<CardContent> where CardContent: Equatable {
         }
         return Int(scoreBonuInterval - matchTimeInterval)
     }
+    
+    mutating func shuffle() {
+        cards.shuffle()
+    }
+    
+    struct Card: Identifiable {
```

Comment:
I would move Card to a standalone file so you can easy check changes for this model